articles/azure-monitor/insights/container-insights-faq.md

```diff
@@ -9,6 +9,46 @@ ms.date: 10/15/2019
 
 This Microsoft FAQ is a list of commonly asked questions about Azure Monitor for containers. If you have any additional questions about the solution, go to the [discussion forum](https://feedback.azure.com/forums/34192--general-feedback) and post your questions. When a question is frequently asked, we add it to this article so that it can be found quickly and easily.
 
+## I dont see 'Image' and 'Name' fields populated for my logs in ContainerLog table
+
+Starting with the agent version ciprod12042019 (and later) , by default, these two fields are not populated for every logline, to save customer costs on log data collected. There are two options to get these field values -

```

Comment:
Suggest re-writing beginning sentences to - Starting with agent version ciprod12042019 and later, by default these two properties are not populated for every log line to minimize cost incurred on log data collected.  There are two options to query the table that include these properties with their values: